include/mbedtls/platform.h

```diff
@@ -116,12 +118,13 @@ extern void * (*mbedtls_calloc)( size_t n, size_t size );
 extern void (*mbedtls_free)( void *ptr );
 
 /**
- * \brief   Set your own memory implementation function pointers
+ * \brief   This function allows configuring custom memory-implementation 
```

Comment:
Preexisting: this shouldn't mention pointers. They're the means, not the goal. This applies to the similar descriptions below.

Preexisting: “memory implementation” is unusual terminology and hard to understand. A (c)alloc function and a free function are memory management functions.

> This function allows configuring custom memory management functions.

Reply:
fixed